### PR TITLE
follow/unfollow

### PR DESCRIPTION
**Pull request by vuhuykhoi**
Screenshot from 2019-07-28 12-41-36
Follow/Unfollow User
Show Followers/Following
Fix footer
Fix home page
Add fake data to seed.db

### CODE REVIEW

**Comment by vuhuykhoi** [comment]
ready

File: app/controllers/relationships_controller.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class RelationshipsController < ApplicationController
+  before_action :logged_in_user
+
+  def create
+    @user = User.find params[:followed_id]
+
+    current_user.follow(@user)
```
Comment (datcnt-0874) [inline app/controllers/relationships_controller.rb]:
```suggestion
    current_user.follow @user
```

File: app/controllers/relationships_controller.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    respond_to do |format|
+      format.html{redirect_to @user}
+      format.js
+    end
+  end
+
+  def destroy
+    @user = Relationship.find(params[:id]).followed
+
+    current_user.unfollow(@user)
```
Comment (datcnt-0874) [inline app/controllers/relationships_controller.rb]:
```suggestion
    current_user.unfollow @user
```
có trường hợp ko tìm thấy @user ko nhỉ?

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -1,6 +1,7 @@
 class UsersController < ApplicationController
   before_action :logged_in_user, only: %i(index edit update destroy)
-  before_action :load_user, only: %i(show edit update destroy)
+  before_action :load_user,
+    only: %i(show edit update following followers)
   before_action :correct_user, only: %i(edit update)
   before_action :admin_user, only: %i(destroy)
```
Comment (datcnt-0874) [inline app/controllers/users_controller.rb]:
```suggestion
  before_action :admin_user, only: :destroy
```

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -49,20 +50,25 @@ def destroy
     redirect_to users_url
   end
 
+  def following
+    @title = t "label.following"
+    @users = @user.following.page(params[:page])
```
Comment (datcnt-0874) [inline app/controllers/users_controller.rb]:
```suggestion
    @users = @user.following.page params[:page]
```

File: app/controllers/users_controller.rb [hunk 5]
```diff
@@ -49,20 +50,25 @@ def destroy
...
+    render :show_follow
+  end
+
+  def followers
+    @title = t "label.followers"
+    @users = @user.followers.page(params[:page])
```
Comment (datcnt-0874) [inline app/controllers/users_controller.rb]:
```suggestion
    @users = @user.followers.page params[:page]
```

**Comment by vuhuykhoi** [comment]
ready

**Pull request merged**